ajax/libs/overlayscrollbars/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "overlayscrollbars",
+  "filename": "OverlayScrollbars.min.js",
+  "license": "MIT",
+  "description": "A javascript scrollbar plugin which hides the native scrollbars and provides custom styleable overlay scrollbars, but keeps the native functionality and feeling.",
+  "homepage": "https://kingsora.github.io/OverlayScrollbars",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/KingSora/OverlayScrollbars"
+  },
+  "author": {
+    "name": "Rene Haas | KingSora"
+  },
+  "keywords": [
+    "overlayscrollbars",
+    "custom",
+    "scrollbar",
+    "scrollbars",
+    "scroll",
+    "frontend"
+  ],
+  "npmName": "overlayscrollbars",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "js/jquery.overlayScrollbars.js",
+        "js/jquery.overlayScrollbars.min.js",
+        "js/OverlayScrollbars.js",
+        "js/OverlayScrollbars.min.js",
+        "css/OverlayScrollbars.css",
+        "css/OverlayScrollbars.min.css"
```

Comment:
Let's use [glob](https://github.com/isaacs/node-glob) pattern to fetch all above files:
```js
"+(css|js)/+(*o|O)verlayScrollbars*"
```